docs/source/general/callback.rst

```diff
@@ -0,0 +1,41 @@
+########
+Callback
+########
+
+.. _callback:
+
+**************
+Flash Callback
+**************
+
+``FlashCallback`` are an extension of PyTorch Lightning :class:`~pytorch_lightning.callbacks.Callback`.
```

Comment:
```suggestion
:class:`~flash.data.callback.FlashCallback` are extensions of the PyTorch Lightning :class:`~pytorch_lightning.callbacks.Callback`.
```

Reply:
The lightning link doesn't work, not sure why. Can you maybe find out or replace with a link?

Reply:
Done !